www/js/con-author.js

```diff
@@ -821,29 +823,32 @@ define([
 
 		addStudentNode: function(nodeid){
 			this.removeStudentNode(nodeid);
-			var currentNode = this._model.authored.getNode(nodeid);
```

Comment:
Sorry Dharani, this is too much overhead for a very small thing. Creating a complete node again everytime. You don't have to just make it work. You have to do it optimally. The overhead that I am talking about is updating the node everytime in the model, going through fixing the equations and so on. This shouldnt be done.